App/Components/HomeScreen.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react';
+import {View, Text, StyleSheet,FlatList,TouchableOpacity} from 'react-native';
+
+
+const style = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#5A6978',
+  },
+  countryContainer: {flex: 0.15, marginStart:20,marginEnd:20, backgroundColor: '#5A6978'},
+  countryBrand: {
+    width: 72,
+    height: 55,
+    marginTop: 15,
+    marginStart: 20,
+    backgroundColor: '#13CE66',
+  },
+  countryName: {
+    position: 'absolute',
+    marginTop: 25,
+    marginLeft: 130,
+    fontSize: 20,
+    color: 'white',
+  },
+});
+
+const list = (item) => {
+  return (
+    <TouchableOpacity underlayColor={'white'}>
+      <View style={style.container}>
+        <View style={style.countryContainer}>
+          <View style={style.countryBrand} />
+          <Text style={style.countryName}> {item.name} </Text>
+        </View>
+      </View></TouchableOpacity>
+  );
+};
+class HomeScreen extends Component {
+  render() {
+    return (
+
+      <FlatList
+        style={style.container}
+        data={[
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+          {name: 'Brasil'},
+        ]}
```

Comment:
O RN está reclamando que está faltando as keys. Sugiro deixar uma key única para cada item, e adicionar um `keyExtractor` que retorna essa key.

Reply:
added  'id' field in each object of the array on the Data props. keyExtractor = {item => item.id}